client/src/Components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Story } from '@storybook/react';
+import Input, { InputProps } from '../Input/Input';
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+  title: 'Input',
+  component: Input,
+  decorators: [(Story: any) => <Story />],
+};
+
+const Template: Story<InputProps> = (args) => <Input {...args} />;
+
+export const PrimaryInput = Template.bind({});
+PrimaryInput.args = {
```

Comment:
Przenieśmy to przed wywyłaniami komponentu(przed 11 linijke aktualnie)
Zamiast args nazwijmy to np. basePropsData

Reply:
Tutaj powinno być basePropsData